app/views/jobs/feed.atom.builder

```diff
@@ -7,7 +7,17 @@ atom_feed language: 'pt-BR' do |feed|
 
     feed.entry(item) do |entry|
       entry.title item.title
-      entry.content item.description, type: 'html'
+      entry.company item.company
+      entry.contact item.email
+      entry.description item.description
+      entry.skills item.skills if item.skills.present?
+      entry.location item.location if item.location.present?
+      entry.modality item.modality if item.modality.present?
+      entry.website item.website if item.website.present?
+      entry.salary item.salary if item.salary.present?
+      entry.url item.url if item.url.present?
```

Comment:
Se alguem quiser filtrar o feed somente com vagas presencial, clt e senior essas tags facilitam.

Um exemplo de feed:

<entry>
    <id>tag:localhost,2005:Job/2</id>
    <published>2015-07-06T17:01:32Z</published>
    <updated>2015-07-06T17:01:32Z</updated>
    <link rel="alternate" type="text/html" href="http://localhost:3000/jobs/2-desenvolvedor-ruby-on-rails-senior"/>
    <company>Empresa A</company>
    <contact>rodrigomonteirof@gmail.com</contact>
    <contract>clt</contract>
    <description>Quisque semper molestie lobortis. Aliquam interdum quis dolor dictum condimentum. Aliquam erat volutpat. Nulla a eros elementum massa pulvinar vestibulum. Pellentesque ullamcorper libero nec eros bibendum, interdum ultricies eros placerat. Donec auctor justo in dapibus aliquam. Nam in nisi justo. Quisque malesuada vulputate congue. Curabitur ullamcorper arcu a magna efficitur malesuada fringilla a libero. Cras cursus nisl et varius commodo. Pellentesque et massa at odio sollicitudin ornare. Sed pulvinar ante lectus, eget porttitor purus facilisis id. Aenean ullamcorper, nunc et porta fermentum, erat urna vehicula enim, nec pellentesque velit leo ut dolor. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Praesent id felis hendrerit, ornare nibh nec, volutpat est. Proin sollicitudin urna in neque ornare, at tristique est varius.</description>
    <location>São Paulo - SP</location>
    <modality>presencial</modality>
    <salary_label>senior</salary_label>
    <skills>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur nec fringilla mi. Pellentesque fermentum ligula lacus, vitae convallis elit egestas non. Phasellus auctor nisl a diam porttitor rhoncus. Quisque tempor velit vitae nibh venenatis, eu faucibus dui tincidunt. Sed non egestas leo, ut faucibus metus. Nullam vel maximus mauris. Maecenas dictum ante arcu, ut auctor massa pellentesque quis. Pellentesque semper enim in lectus sollicitudin euismod. Sed eleifend diam a scelerisque ullamcorper. Integer lacus ipsum, faucibus quis luctus id, iaculis nec eros. Praesent ac imperdiet nisl. Aenean efficitur nunc vitae gravida luctus. Maecenas at nibh lectus. In eleifend sapien non ipsum scelerisque faucibus.</skills>
    <title>Desenvolvedor Ruby on Rails Senior</title>
    <website>http://www.goaltracker.com</website>
    <content type="html">      &lt;h1&gt;
        Desenvolvedor Ruby on Rails Senior
      &lt;/h1&gt;
      &lt;p&gt;
        &lt;span class="job-company"&gt;
          &lt;a href="http://www.goaltracker.com"&gt;Empresa A&lt;/a&gt;
&lt;/span&gt;        &lt;span class="job-location"&gt;São Paulo - SP/Presencial&lt;/span&gt;
        &lt;span class="job-date"&gt;06 de Julho de 2015, 17:01&lt;/span&gt;
        &lt;span format="long" class="job-contract-type-clt"&gt;Contratação: CLT&lt;/span&gt;
        &lt;span format="long" class="job-salary-senior"&gt;Salário: Acima de R$9.000&lt;/span&gt;
      &lt;/p&gt;
    &lt;/div&gt;

    &lt;div class="job-description"&gt;
      &lt;p&gt;Quisque semper molestie lobortis. Aliquam interdum quis dolor dictum condimentum. Aliquam erat volutpat. Nulla a eros elementum massa pulvinar vestibulum. Pellentesque ullamcorper libero nec eros bibendum, interdum ultricies eros placerat. Donec auctor justo in dapibus aliquam. Nam in nisi justo. Quisque malesuada vulputate congue. Curabitur ullamcorper arcu a magna efficitur malesuada fringilla a libero. Cras cursus nisl et varius commodo. Pellentesque et massa at odio sollicitudin ornare. Sed pulvinar ante lectus, eget porttitor purus facilisis id. Aenean ullamcorper, nunc et porta fermentum, erat urna vehicula enim, nec pellentesque velit leo ut dolor. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Praesent id felis hendrerit, ornare nibh nec, volutpat est. Proin sollicitudin urna in neque ornare, at tristique est varius.&lt;/p&gt;

    &lt;/div&gt;

    &lt;div class="well skills"&gt;
      &lt;h4&gt; Habilidades &lt;/h4&gt;
      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur nec fringilla mi. Pellentesque fermentum ligula lacus, vitae convallis elit egestas non. Phasellus auctor nisl a diam porttitor rhoncus. Quisque tempor velit vitae nibh venenatis, eu faucibus dui tincidunt. Sed non egestas leo, ut faucibus metus. Nullam vel maximus mauris. Maecenas dictum ante arcu, ut auctor massa pellentesque quis. Pellentesque semper enim in lectus sollicitudin euismod. Sed eleifend diam a scelerisque ullamcorper. Integer lacus ipsum, faucibus quis luctus id, iaculis nec eros. Praesent ac imperdiet nisl. Aenean efficitur nunc vitae gravida luctus. Maecenas at nibh lectus. In eleifend sapien non ipsum scelerisque faucibus.
&lt;/div&gt;
    &lt;div class="row do-you-like-it"&gt;

      &lt;div class="col-xs-12 col-sm-12 col-md-1 col-lg-1"&gt;
        &lt;img src="/assets/logo-6192ac44030c8a2cabddc9f99ef287d9.png" alt="Logo" /&gt;
      &lt;/div&gt;

      &lt;div class="col-xs-12 col-sm-12 col-md-11 col-lg-11"&gt;
        &lt;h4&gt; Gostou da vaga? &lt;/h4&gt;
        &lt;p&gt;
          Entre em contato com o responsável pelo email &lt;span class="email"&gt;rodrigomonteirof@gmail.com&lt;/span&gt; e caso seja contratado deixe-nos saber!
        &lt;/p&gt;
      &lt;/div&gt;
</content>
    <author>
      <name>Empresa A</name>
    </author>
  </entry>